checks/raw/fuzzing_test.go

```diff
@@ -394,6 +394,116 @@ func Test_checkFuzzFunc(t *testing.T) {
 			},
 			fileContent: "import Test.Hspec",
 		},
+		{
+			name:     "Elixir QuickCheck through PropCheck",
+			want:     true,
+			fileName: []string{"Test.exs"},
+			langs: []clients.Language{
+				{
+					Name:     clients.Elixir,
+					NumLines: 50,
+				},
+			},
+			fileContent: "use PropCheck, default_opts: &PropCheck.TestHelpers.config/0",
+		},
+		{
+			name:     "Elixir QuickCheck through StreamData",
+			want:     true,
+			fileName: []string{"Test.exs"},
+			langs: []clients.Language{
+				{
+					Name:     clients.Elixir,
+					NumLines: 50,
+				},
+			},
+			fileContent: "use ExUnitProperties",
+		},
+		{
+			name:     "Elixir with no property-based testing",
+			want:     false,
+			fileName: []string{"NoPropTest.exs"},
+			wantErr:  true,
+			langs: []clients.Language{
+				{
+					Name:     clients.Haskell,
```

Comment:
Thanks a lot @voltone ! It was a good catch, because we were testing that no property-based testing happens, and based on the patterns, this surely should not have been detected by Haskell either :) 
Thanks!